ISO Documentation Edits

lindsmith opened the pull request:
Documentation edits from the ISO review process. Main edits include:
- Addition of '(list)' label where appropriate
- Addition of Interpretation clarification to match the one added to the element overview figure
- Changes to the description column in some tables
- 'repeated' in Status column of diagnosis and genomic interpretation changed to 'optional'

julesjacobsen commented on docs/disease.rst:
```diff
@@ -58,7 +58,7 @@ disease_stage
 
 This attribute is used to describe the stage of disease. If the disease is a cancer, this attribute describes
-the extent of cancer development, typically including an AJCC stage group (i.e., Stage 0, I-IV), though other staging
+the extent of cancer development, typically including an AJCC (i.e., Stage 0, I-IV), though other staging
```
why remove the 'stage group'? AJCC is the acronym of the American Joint Committee on Cancer.

lindsmith replied:
@julesjacobsen Apologies, this is an error! I checked the ISO doc and it's still present. A commenter suggested to add TNM to this sentence, which I did add, but then later removed because of the TNM_finding field below. I may have accidently removed 'stage group' with it. Will fix this.

julesjacobsen commented on docs/phenopacket.rst:
```diff
@@ -25,35 +25,35 @@ describe the phenotypic abnormalities directly associated with an extirpated or
+      - :ref:`rstfile` (list)
       - optional
-      - VCF or other high-throughput sequencing files
+      - High-throughput sequencing files (ie. VCF)
```
Replace 'i.e.' with 'e.g.' as in 'for example'. To quote https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference

> i.e. is an abbreviation for the phrase id est, which means "that is." I.e. is used to restate something said previously in order to clarify its meaning. E.g., on the other hand, is short for exempli gratia, which means "for example." E.g. is used before an item or list of items that serve as examples for the previous statement.

lindsmith replied:
Done

julesjacobsen commented on docs/schema.rst:
```diff
@@ -25,7 +25,9 @@ and machine-readable, and to inter-operate with standards being developed in org
 ~~~~~~~~
 
-The diagram below shows an overview of the schema elements.
+The diagram below shows an overview of Phenopackets schema elements and relationships. The orange circles represent an Interpretation and its sub-elements. Blue circles are the top-level elements used to describe an individual or group of individuals, and the green circles are the building block components. Lines between elements indicate composition.
```
Please update this using the https://raw.githubusercontent.com/phenopackets/phenopacket-schema/v1.1/docs/schema.rst to work off. Sorry, we slightly deviated on that one!

lindsmith replied:
@julesjacobsen Updated - please double-check that it's what you were looking for!

julesjacobsen commented on docs/biosample.rst:
```diff
@@ -71,11 +71,11 @@ genomic array as well as RNA-seq experiments) may refer to the same Biosample.
-     - :ref:`rstfile`
+     - :ref:`rstfile` (list)
      - optional
      - list of high-throughput sequencing files derived from the biosample
```
Capitalise 'L' in 'list'

lindsmith replied:
@julesjacobsen Done. I'll review the other tables as well, I think there are some instances where tables are all lowercase and others are capitalized. I'll make them all capitalized for consistency.

julesjacobsen reviewed (commented):
Thanks for the update - I've added a few comments to the changes.

pnrobinson reviewed (commented):
Seems all fine. Capitalize List as suggested by @julesjacobsen . Not sure about the capitalization

lindsmith commented:
@pnrobinson @julesjacobsen Thanks to you both for reviewing! I have gone through and capitalized all table descriptions for consistency... re: Peter's comment, should I revert some of those? Not sure about the correct format.
Please note, you'll see a few more commits to Resource and Variant. In Resource, I changed the table descriptions from examples to written descriptions (there's already an example below the table). Also rearranged the order of the table elements to match order of the example following the table. Same thing for Variant - I added written descriptions to some of the tables. Let me know if you have any suggested edits for those!
